windows/rngallery/MainPage.xaml

```diff
@@ -8,12 +8,9 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-    <Grid>
-        <Canvas x:Name="RNPrintCanvas" Opacity="0" />
-        <react:ReactRootView 
-            x:Name="ReactRootView"
-            ComponentName="rngallery"
-            Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
-            MinHeight="400"/>
-    </Grid>
+    <react:ReactRootView 
```

Comment:
Another automatic change.
@jonthysell  Should I revert?

The app loads fine with these changes.